### PR TITLE
[RND-247] Kafka to OpenSearch - MongoDB

### PR DESCRIPTION
**Pull request by andonyns**
- Separate kafka setup to its own backend folder.
- Add sink connector for OpenSearch.
- Add Java project for transformation of index into OpenSearch.
- Add Dockerfile for kafka-connect with custom transformations.
- Add meadowlark-net to be able to communicate between mongo -> opensearch -> kafka.

### CODE REVIEW

File: Meadowlark-js/backends/meadowlark-kafka-stream/docker/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
+version: '3'
```
Comment (stephenfuqua) [inline Meadowlark-js/backends/meadowlark-kafka-stream/docker/docker-compose.yml:1]: Glad to see you've moved this out of the MongoDB startup into its own area.

File: docker/kafka/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+FROM gradle:8.2.1-jdk-focal@sha256:78b95c865c1bcb50ced01501e6a9c6c9bcc181be93c1ea723ff4b58b2f7e9a96 AS build
+COPY --chown=gradle:gradle /ed-fi-kafka-connect-transforms /home/gradle/src
+WORKDIR /home/gradle/src
+RUN gradle installDist --no-daemon
+
+FROM debezium/connect:2.3@sha256:dfa59c008a03f45c7b286d2874f2e6dbe04f3db6f26b6f01806c136abb07381a
```
Comment (stephenfuqua) [inline docker/kafka/Dockerfile:11]: Nice use of layering for optimization 😊

File: docker/kafka/ed-fi-kafka-connect-transforms/LICENSE
Comment (stephenfuqua) [inline docker/kafka/ed-fi-kafka-connect-transforms/LICENSE:1]: Do we really need this? Already have a license file at the root of the project.

File: docker/kafka/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
+# Licensed to the Ed-Fi Alliance under one or more agreements.
+# The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+# See the LICENSE and NOTICES files in the project root for more information.
+
+FROM gradle:8.2.1-jdk-focal@sha256:78b95c865c1bcb50ced01501e6a9c6c9bcc181be93c1ea723ff4b58b2f7e9a96 AS build
```
Comment (stephenfuqua) [inline docker/kafka/Dockerfile:6]: I had to look this up... confirmed that's a new JDK 17 👍

File: docker/kafka/ed-fi-kafka-connect-transforms/LICENSE
Comment (andonyns) [reply docker/kafka/ed-fi-kafka-connect-transforms/LICENSE:1]: Removed

**Review by bradbanister — APPROVED** [review]

**Pull request merged**